### DISCUSSION TITLE
Logging last seen for device

### DISCUSSION THREAD

**Discussion by avdept** [Koenkk/zigbee2mqtt#11055] [Devices]
I was kinda interested if its possible to log somehow last seen timestamp, so I could use it in my automations to track "unresponsive" devices and sensors. Is something like this exists already in z2m?

**Comment by Koenkk** [comment]
https://www.zigbee2mqtt.io/guide/configuration/device-availability.html

**Reply by avdept** [reply]
Thanks for sharing, will have a look. But seems to be what I was looking for!
